### PR TITLE
PLT-571 Restricted file uploads on iOS Chrome and the iOS app to work around iOS bugs

### PR DESCRIPTION
**Pull request by hmhealey**
There's bugs in the file uploading on iOS so I had to turn off the broken stuff in Chrome and the mobile app. Chrome on iOS can't upload videos (they contain no data) so I turned that off and the web view that is used in the iOS app can't upload videos (same issue) when multiple upload is enabled.

### CODE REVIEW

File: web/react/components/file_upload.jsx [hunk 1]
```diff
@@ -280,7 +293,7 @@ export default class FileUpload extends React.Component {
                     ref='fileInput'
                     type='file'
                     onChange={this.handleChange}
-                    multiple='true'
+                    {...inputProps}
```
Comment (coreyhulen) [inline web/react/components/file_upload.jsx]: It's the `...` necessary?  Lets try to avoid this kind of complexity.

**Comment by jwilander** [comment]
+1, other than CH's comment

File: web/react/components/file_upload.jsx (hunk 1 above)
Comment (hmhealey) [inline web/react/components/file_upload.jsx]: Nope, I can get rid of it. I had that in there from some other stuff I was doing with the input while testing and just never took it out.

**Comment by hmhealey** [comment]
Made those fixes @coreyhulen

**Pull request merged**